readme.txt

```diff
@@ -113,6 +113,7 @@ There are opportunities for developers at all levels to contribute. [Learn more
 * Dashboard: display a notice when a site uses conflicting plans.
 * Map Block: improve the calculation and persistence of maps' center points.
 * Map Block: improve the generation of an access token on WordPress.com sites.
+* Map Block: update Mapbox GL library to opt into map load based billing.

```

Comment:
```suggestion
* Map Block: support opt-in for usage-based billing.
```

Reply:
I like the shorter version, but I also like mentioning that this is about the Mapbox library; the billing itself isn't related to Jetpack really.